### DISCUSSION TITLE
Can I run database in RDS?

### DISCUSSION THREAD

**Discussion by mchtter** [supabase/supabase#4993] [Questions]
I set "postgres://postgres:${POSTGRES_PASSWORD}@${DB_HOST}:5432/postgres?search_path=auth" in docker-compose.yml and I connected RDS DB but catch Auth and Storage error in localhost:3000 screen and I see superuser problem in terminal.

I try create default permission supabase_admin role but I can't do this. Because I haven't superuser permission in RDS. Is this setting required for supabase db connection?

Can I use RDS DB with supabase without superuser?

What should I do for this?

**Comment by igorgrin** [comment]
Any updates on this question please?

**Reply by nirberko** [reply]
Im trying to run the DB also on RDS and unable to do it, did you manage to solve the issue?

**Comment by johndpope** [comment]
i made some progress here - https://github.com/johndpope/rds-supabase 
in the rds_schema - replace your_rds_admin with your master admin account.

some notes here at bottom
https://github.com/orgs/supabase/discussions/34342
